tool/actions-plan-preview/main.go

```diff
@@ -95,22 +103,28 @@ func main() {
 			body,
 		)
 		if err != nil {
-			log.Fatal(err)
+			log.Println(err)
+			return err
```

Comment:
IMO, if this error is returned, and the caller prints again, it's redundant. If we don't have specific logic based on the error, then instead of returning the error, we should return the existing code. WDYT?

Reply:
I'm unsure about that.

Initially, I wrote as you said and thought it was not the standard go code to check whether the return value was zero.
Secondly, I wrote just return the error from the `doComment`, printed it on the caller side, and was a bit nervous about whether I had written all the logging.
Finally, I wrote logging in the `doComment`, returned an error, and didn't print it on the caller side.

Reply:
I see. Thanks for the clarification. Because I saw many places that perform "if error, then return 1, else return 0," I think it could be better to emit it as well. This means we logged the error and returned 0 and 1 from this doComment, and on the caller side, only returning doComment is enough.